### PR TITLE
Added building instructions for vector testing example to readme

### PR DESCRIPTION
**Pull request by sovesti**
Build instructions improvement

### CODE REVIEW

**Comment by sovesti** [comment]
Should have been in the previous #1 PR

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # loft-docs
 LOFT Documentation
+
+## Vector tests
+
+### Building
+
+```
+cd examples/vectorTests
+cmake -G "<generator>" -S "" -B "build"
```
Comment (zelenyhleb) [inline README.md]: Please add some note about generator. What is it? How can I determine, which one is applicable for me in this case.

**Review by zelenyhleb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ruspl-afed) [reply README.md]: Well, it is quite a laconic note, but looks good.

**Review by ruspl-afed — APPROVED** [review]
Thanks @sovesti 
Looking forward to see the CI action in action.

**Review by zelenyhleb — APPROVED** [review]
Nice!

**Pull request merged**